src/contracts/payloads/PayloadsControllerCore.sol

```diff
@@ -133,7 +138,7 @@ abstract contract PayloadsControllerCore is
     );
 
     uint256 executionTime = payload.queuedAt + payload.delay;
-    require(block.timestamp > executionTime, Errors.TIMELOCK_NOT_FINISHED);
+    require(block.timestamp >= executionTime, Errors.TIMELOCK_NOT_FINISHED);
```

Comment:
why was the = added? It was left with strict > to follow the property that execution can only happen on time strictly after execution time

Reply:
Added to leave opportunity set zero delay and execute changes immediately. Was discussed with @eboadom. May be we need to let him resolve this discussion

Reply:
yes, as this changes main PayloadsController functionality, plus would require also changes on main tests, properties etc.
In my opinion should not be changed. A I think it also doesnt make sense to have 0 delay

Reply:
My point was on the permissioned layer, not on the core. But considering that overloading the creation method only for that would be pretty burdensome, better to skip and keep the strict `>`.
Ofc modifying the child too.